apis/python/devtools/ingestor

```diff
@@ -90,11 +92,36 @@ select `relative=True`. (This is the default.)
         nargs=1,
     )
     parser.add_argument(
+        # Given a tiledb.Config object `cfg`, you can do `cfg.save("mycfg.txt")`.
+        # It'll look like this:
+        #
+        # ...
+        # config.logging_level 0
+        # filestore.buffer_size 104857600
+        # rest.curl.buffer_size 524288
+        # ...
+        #
+        # Then you can edit it if you like. Then you can have this script read it
+        # using `--tiledb-config mycfg.txt`.
         "--tiledb-config",
         help="TileDB config data from `config.save()`",
         type=str,
         default=None,
     )
+    parser.add_argument(
+        # Given a PlatformConfig object -- just a nested dict -- named `platform_config` you can do
+        #
+        #   import json
+        #   with open("pc.json", "w") as F:
+        #     json.dump(platform_config, F)
+        #
+        # Then you can edit it if you like. Then you can have this script read it
+        # using `--platform-config pc.json`.
+        "--platform-config",
+        help="Platform-config data as JSON",
+        type=str,
```

Comment:
For things intended to be opened as files, consider using https://docs.python.org/3/library/argparse.html#argparse.FileType